src/test/groovy/com/github/vkorobkov/jfixtures/loader/FixtureValueTest.groovy

```diff
@@ -61,4 +61,19 @@ class FixtureValueTest extends Specification {
         _ | 50
         _ | 100500L
     }
+
+    def "#toString returns uppercased null value"() {
+        expect:
+        new FixtureValue(null).toString() == "NULL"
```

Comment:
Let's also check that the fixture type is `AUTO` so the `NULL` value will not be turned to string in output SQL